pkg/provisionerapi/worker.go

```diff
@@ -0,0 +1,65 @@
+package provisionerapi
+
+import (
+	"net/http"
+
+	"github.com/gin-gonic/gin"
+	"github.com/iver-wharf/wharf-cmd/pkg/provisioner"
+	"github.com/iver-wharf/wharf-core/pkg/ginutil"
+	"k8s.io/client-go/rest"
+)
+
+type workerModule struct {
+	kubeConfig *rest.Config
+	p          provisioner.Provisioner
```

Comment:
nit: For field names I suggest longer names. Such as `prov`, or the complete `provisioner` word.

Reply:
Went with `prov`.